public/js/reader.js

```diff
@@ -52,6 +54,16 @@ const readerComponent = () => {
 					if (savedMargin) {
 						this.margin = savedMargin;
 					}
+
+					// Preload Images
+					this.preloadLookahead = +localStorage.getItem('preloadLookahead') ?? 3;
```

Comment:
When the key doesn't exist in local storage, `getItem` returns `null`, but the `+` sign turns it into `0` (because `+null` evaluates to 0). I think we can remove the plus sign here?

Reply:
I see! Since `getItem()` returns a string and I want to keep `this.preloadLookahead` as a number, I'll parenthesize them.